gammapy/data/data_store.py

```diff
@@ -711,3 +713,59 @@ def make_psf(self, position, energy=None, theta=None):
         psf_tot = EnergyDependentTablePSF(energy=energy, offset=theta, exposure=exposure,
                                           psf_value=psf_value.T)
         return psf_tot
+
+    def make_mean_edisp(self, position, e_true, e_reco, low_reco_threshold=None, high_reco_threshold=None):
+        r"""Make mean rmf for a given position and a set of observations.
+
+        Compute the mean rmf of a set of observations j at a given position
+
+        The stacking of :math:`j` observations is implemented as follows.  :math:`k`
+        and :math:`l` denote a bin in reconstructed and true energy, respectively.
+
+        .. math::
+
+            \epsilon_{jk} =\left\{\begin{array}{cl} 1, & \mbox{if
+                bin k is inside the energy thresholds}\\ 0, & \mbox{otherwise} \end{array}\right.
+
+            \overline{\mathrm{edisp}}_{kl} = \frac{\sum_{j} \mathrm{edisp}_{jkl}
+                \cdot \mathrm{aeff}_{jl} \cdot t_j \cdot \epsilon_{jk}}{\sum_{j} \mathrm{aeff}_{jl}
+                \cdot t_j}
+
+        Parameters
+        ----------
+        position : `~astropy.coordinates.SkyCoord`
+            Position at which to compute the mean EDISP
+        e_true : `~gammapy.utils.energy.EnergyBounds`
+            True energy axis
+        e_reco : `~gammapy.utils.energy.EnergyBounds`
+            Reconstructed energy axis
+        low_reco_threshold : `~gammapy.utils.energy.Energy`
+            low energy threshold in reco energy, default 0.002 TeV
+        high_reco_threshold : `~gammapy.utils.energy.Energy`
+            high energy threshold in reco energy , default 150 TeV
+        Returns
+        -------
+        stacked_rmf: `~gammapy.irf.EnergyDispersion`
+            Stacked RMF for a set of observation
+        """
+
+        list_arf = list()
+        list_edisp = list()
+        list_livetime = list()
+        if not low_reco_threshold:
```

Comment:
We looked at this in detail and decided that it's OK / best to use quantities as default keyword arguments.
So here you don't have to put `None` and then this if statement, please put `low_reco_threshold=Energy(0.002, "TeV")` directly in the function signature.